shared/modules/types.go

```diff
@@ -48,6 +48,11 @@ type PersistenceConfig interface {
 	GetBlockStorePath() string
 	GetTxIndexerPath() string
 	GetTreesStoreDir() string
+	GetMaxConns() int32
+	GetMinConns() int32
+	GetMaxConnLifetime() uint64
```

Comment:
For these three, can you add a postfix explaining the time parameter? The code speaks for itself then

E.g.

`s/GetMaxConnLifetime/GetMaxConnSec`
`s/GetMaxIdleTime/GetMaxIdleSec`

Reply:
Yep sure, will do